### PR TITLE
Move to go modules

### PR DESCRIPTION
**Pull request by chenrui333**
Issue tracker, https://github.com/containerd/containerd/issues/3031

### CODE REVIEW

**Comment by chenrui333** [comment]
```
$ go test -race -covermode=atomic -c
# github.com/containerd/aufs [github.com/containerd/aufs.test]
./aufs.go:246: undefined: snapshots.WalkFunc
./aufs.go:252: too many arguments in call to storage.WalkInfo
./aufs.go:441: undefined: unix.Mount
```

**Comment by AkihiroSuda** [comment]
should be consistent with containerd/zfs#26

**Comment by chenrui333** [comment]
@AkihiroSuda FYI, I will spend more time on this later today or tomorrow. :)

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module github.com/containerd/aufs
+
+go 1.13
+
+require (
+	github.com/Microsoft/hcsshim v0.8.7 // indirect
+	github.com/containerd/containerd v1.3.2
```
Comment (AkihiroSuda) [inline go.mod]: vendor master as in https://github.com/containerd/zfs/pull/26

Comment (chenrui333) [reply go.mod]: Keep the deps consistent with zfs now.

Comment (chenrui333) [reply go.mod]: Works great, thanks!

**Review by AkihiroSuda — APPROVED** [review]

**Review by fuweid — APPROVED** [review]
LGTM

File: .travis.yml [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
 
 before_script:
     - pushd ..; git clone https://github.com/containerd/project; popd
 
 script:
     - DCO_VERBOSITY=-q ../project/script/validate/dco
     - ../project/script/validate/fileheader ../project/
-    - go test -race -covermode=atomic -c
-    - sudo ./aufs.test -test.root -test.v -test.coverprofile=coverage.txt
+    - go test -v -race -covermode=atomic -coverprofile=coverage.txt ./...
```
Comment (AkihiroSuda) [inline .travis.yml:17]: this part broke the CI, @chenrui333 could you revert this part?

```
=== RUN   TestAufs
--- SKIP: TestAufs (0.00s)
    helpers_unix.go:41: skipping test that requires root
```

Comment (chenrui333) [reply .travis.yml:17]: Got it, I will fix it now!

Comment (chenrui333) [reply .travis.yml:17]: PR, https://github.com/containerd/aufs/pull/19

**Pull request merged**